### ISSUE TITLE
Adding local files to playlist

### ISSUE THREAD

**Issue by Keith94** [digimezzo/dopamine-windows#266]
I tried to drag and drop a folder/file into a new playlist directly, but it did not add the tracks. Will such a thing ever be supported, or can you explain how it's supposed to work?

**Comment by Riajyuu** [comment]
see what @digimezzo said in my issue #263

**Comment by Keith94** [comment]
It only seems to address "playlist files" not necessarily the same as "local files", correct me if I'm wrong.

I would love to have a similar feature as "Autoplaylists" (from foobar2000) -- i.e. playlists which points to a custom folder and refreshes automatically.

**Comment by Riajyuu** [comment]
what i mean is: files can be added only by playlists on that page. Also tracks not in album collection will be also excluded. I don't actually understand why @digimezzo  designs it like this. 
As for as "Autoplaylists", AIMP also has similar features.

**Comment by digimezzo** [comment]
There is not much to understand about the design: I just had to start somewhere. An over time, adding extra features, made it inconsistent. The original idea was to create playlists from files which are in the collection. That worked well, and supported track rating, as I could save ratings in the collection database. Then, there was a request to open m3u files. There was a problem with adding that feature: track which are not in the database, can be added, but rating cannot be saved. So, what to do? I decided to limit adding only files which are in the collection. Bad choice apparently. After a lot of feedback, I decided to redo playlist support from scratch, see #124. That'll come in version 1.3, but will take some time to implement.
